Makefile

```diff
@@ -15,7 +15,7 @@ build: generate vendor-sync
 all: docker-build
 
 generate:
```

Comment:
This generate step is no longer necessary, since the `iprange` repo has the latest generated code.